chainer/functions/array/moveaxis.py

```diff
@@ -0,0 +1,114 @@
+import six
+
+from chainer import cuda
+from chainer import function_node
+from chainer.utils import type_check
+
+
+def _normalize_axis_tuple(axis, ndim, xp):
+    if xp.isscalar(axis):
+        axis = (axis,)
+
+    ret = []
+    for ax in axis:
+        ret.append(ax % ndim)
+    return ret
+
+
+def _moveaxis(a, source, destination, xp):
+    if hasattr(xp, 'moveaxis'):
+        return xp.moveaxis(a, source, destination)
+    if all(isinstance(a, int) for a in source):
```

Comment:
`all` → `not all`

Reply:
Thanks! I fixed here and addressed other your comments.